src/runtime/runtime_tkey_baremetal.go

```diff
@@ -0,0 +1,24 @@
+//go:build tkey && !qemu
+
+package runtime
+
+import "device/riscv"
+
+// ticksToNanoseconds converts ticks (at 18MHz) to nanoseconds.
+func ticksToNanoseconds(ticks timeUnit) int64 {
+	return int64(ticks) * 10000
+}
```

Comment:
The comment here doesn't match the source code!
The code suggests one tick is actually 10_000 nanoseconds, or 10µs.

Same for `nanosecondsToTicks`!

Reply:
Oops, I forgot to change the comment from the original values I tried. Modified comment to match the current reality.